install_config/install/advanced_install.adoc

```diff
@@ -776,15 +776,65 @@ openshift_examples_modify_imagestreams=true
 
 |`*openshift_docker_additional_registries*`
 |Specify the additional registry or registries.
+
+|`*openshift_cockpit_deployer_prefix*`
+|Specify the URL and path to the directory where the image is located.
+
+|`*openshift_web_console_prefix*`
+|The prefix for the component images. For example, with
+ifdef::openshift-origin[]
+`openshift/origin-web-console:v3.9`, set prefix `openshift/origin-`.
+endif::[]
+ifdef::openshift-enterprise[]
+`openshift3/ose-web-console:v3.9`, set prefix `openshift3/ose-`.
+endif::[]
+
+|`*openshift_template_service_broker_image_prefix*`
+|
+
+|`*openshift_service_catalog_image_prefix*`
+|For use if you have the service catalog enabled.
+
+|`*ansible_service_broker_image_prefix*`
+|
+
+|`*template_service_broker_prefix*`
+|
+
+|`*openshift_crio_systemcontainer_image_override*`
+|A setting for if you are using CRI-O and if you are using an alternative CRI-O
+system container image from another registry.
 |===
 
 For example:
 ----
 oreg_url=example.com/openshift3/ose-${component}:${version}
 openshift_examples_modify_imagestreams=true
 openshift_docker_additional_registries=example.com
+openshift_cockpit_deployer_prefix='registry.example.com/openshift3/'
+ifdef::openshift-origin[]
+`openshift/origin-web-console:v3.9`, set prefix `openshift/origin-`.
+endif::[]
+ifdef::openshift-enterprise[]
+`openshift3/ose-web-console:v3.9`, set prefix `openshift3/ose-`.
+endif::[]
```

Comment:
should not be something like this?
openshift_web_console_prefix='registry.example.com/openshift3/ose-'